.erb/configs/.eslintrc

```diff
@@ -2,6 +2,8 @@
   "rules": {
     "no-console": "off",
     "global-require": "off",
-    "import/no-dynamic-require": "off"
+    "import/no-dynamic-require": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
Its ignore on a per-line level (ex. `// eslint-disable-next-line`)

Reply:
Fixed :+1: I removed these and added `eslint-disable-next-line`s instead.